bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/handler/SqueezeBoxServerHandler.java

```diff
@@ -680,88 +680,93 @@ private void handleStatusMessage(final String mac, String[] messageParts) {
-                    updatePlayer(listener -> listener.currentPlaylistShuffleEvent(mac, shuffle));
-                }
-                // Parameter Title
-                else if ("title".equals(entry.key)) {
-                    updatePlayer(listener -> listener.titleChangeEvent(mac, entry.value));
-                }
-                // Parameter Remote Title (radio)
-                else if ("remote_title".equals(entry.key)) {
-                    remoteTitle = entry.value;
-                }
-                // Parameter Artist
-                else if ("artist".equals(entry.key)) {
-                    artist = entry.value;
-                }
-                // Parameter Album
-                else if ("album".equals(entry.key)) {
-                    album = entry.value;
-                }
-                // Parameter Genre
-                else if ("genre".equals(entry.key)) {
-                    genre = entry.value;
-                }
-                // Parameter Year
-                else if ("year".equals(entry.key)) {
-                    year = entry.value;
-                }
-                // Parameter artwork_url contains url to cover art
-                else if ("artwork_url".equals(entry.key)) {
-                    artworkUrl = entry.value;
-                }
-                // When coverart is "1" coverid will contain a unique coverart id
-                else if ("coverart".equals(entry.key)) {
-                    coverart = "1".equals(entry.value);
-                }
-                // Id for covert art (only valid when coverart is "1")
-                else if ("coverid".equals(entry.key)) {
-                    coverid = entry.value;
-                } else {
-                    // Added to be able to see additional status message types
-                    logger.trace("Unhandled status message type '{}' (value '{}')", entry.key, entry.value);
+                try {
+                    // Parameter Power
+                    if ("power".equals(entry.key)) {
+                        final boolean power = "1".equals(entry.value);
+                        updatePlayer(listener -> listener.powerChangeEvent(mac, power));
+                    }
+                    // Parameter Volume
+                    else if ("mixer volume".equals(entry.key)) {
+                        final int volume = (int) Double.parseDouble(entry.value);
+                        updatePlayer(listener -> listener.absoluteVolumeChangeEvent(mac, volume));
+                    }
+                    // Parameter Mode
+                    else if ("mode".equals(entry.key)) {
+                        updatePlayer(listener -> listener.modeChangeEvent(mac, entry.value));
+                    }
+                    // Parameter Playing Time
+                    else if ("time".equals(entry.key)) {
+                        final int time = (int) Double.parseDouble(entry.value);
```

Comment:
Perhaps we should not even try to parse "N/A" here, if this is a known string (described in linked issue)? This would be faster than having an exception thrown.

Reply:
I thought about checking explicitly for "N/A". But, since this is the first I've heard of this issue in several years of using/maintaining this binding, I'm just not sure where it came from. So, I thought it best to be defensive in this section of code since there are multiple places where `parseDouble` can fail.

WDYT?

Reply:
And, the net effect of a failure here is that the listener job ends, which requires a bundle restart.